### ISSUE TITLE
Pip packaging has broken rf-notebooks

### ISSUE THREAD

**Issue by vpipkt** [locationtech/rasterframes#157]
Now the docker build doesn't get all the binaries. Within rf-notebooks `pyrasterframes.get_spark_session()` results in 

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-1-9d0dcbc02cbc> in <module>
      2 from pyrasterframes.rasterfunctions import *
      3 
----> 4 spark = pyrasterframes.get_spark_session()

/usr/local/rasterframes/pyrasterframes.zip/pyrasterframes/__init__.py in get_spark_session()
     60     from pyrasterframes.utils import create_rf_spark_session
     61 
---> 62     return create_rf_spark_session()
     63 
     64 

/usr/local/rasterframes/pyrasterframes.zip/pyrasterframes/utils.py in create_rf_spark_session()
     78 def create_rf_spark_session():
     79     """ Create a SparkSession with pyrasterframes enabled and configured. """
---> 80     jar_path = find_pyrasterframes_assembly()
     81 
     82     spark = (SparkSession.builder

/usr/local/rasterframes/pyrasterframes.zip/pyrasterframes/utils.py in find_pyrasterframes_assembly()
     72         raise RuntimeError("""
     73 Expected to find exactly one assembly. Found '{}' instead.
---> 74 Try running 'sbt pyrasterframes/package' first. """.format(jarpath))
     75     return jarpath[0]
```

**Comment by vpipkt** [comment]
Interesting i can see the spark app is up and has the rasterframes.jar in the classpath.

**Comment by metasim** [comment]
Fixed in #158

**Comment by vpipkt** [comment]
I am reopening... I believe we have had a regression on this with my attempt at fixing merge conflicts.  Or maybe there is a non-deterministic behavior depending on which other builds you've done. Or it may be larger than just in notebooks.

Behavior I am now seeing is that in `sbt pyrasterframes/clean && sbt pyrasterframes/package` the whl file does not contain the assembly jar `unzip -l pyrasterframes/target/python/dist/pyrasterframes-0.8.0.dev0-py2.py3-none-any.whl`

Partial output of the sbt package command, note 3rd from last. 

```
copying pyrasterframes/version.py -> build/lib/pyrasterframes
copying pyrasterframes/rf_types.py -> build/lib/pyrasterframes
copying pyrasterframes/__init__.py -> build/lib/pyrasterframes
copying pyrasterframes/context.py -> build/lib/pyrasterframes
copying pyrasterframes/utils.py -> build/lib/pyrasterframes
copying pyrasterframes/rasterfunctions.py -> build/lib/pyrasterframes
copying pyrasterframes/rf_ipython.py -> build/lib/pyrasterframes
creating build/lib/geomesa_pyspark
copying geomesa_pyspark/__init__.py -> build/lib/geomesa_pyspark
copying geomesa_pyspark/types.py -> build/lib/geomesa_pyspark
package init file 'deps/jars/__init__.py' not found (or not a regular file)
running egg_info
creating pyrasterframes.egg-info
writing pyrasterframes.egg-info/PKG-INFO
writing dependency_links to pyrasterframes.egg-info/dependency_links.txt
writing requirements to pyrasterframes.egg-info/requires.txt
writing top-level names to pyrasterframes.egg-info/top_level.txt
writing manifest file 'pyrasterframes.egg-info/SOURCES.txt'
reading manifest file 'pyrasterframes.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no previously-included files matching '*.py[cod]' found anywhere in distribution
warning: no previously-included files matching '__pycache__' found anywhere in distribution
warning: no previously-included files matching '.DS_Store' found anywhere in distribution
warning: no files found matching '*.jar' under directory 'deps/jars'
writing manifest file 'pyrasterframes.egg-info/SOURCES.txt'
running bdist_wheel
```
